packages/mco/components/mco-dashboard/data-policy/cluster-app-card/argo-application-set.tsx

```diff
@@ -219,7 +223,7 @@ export const ReplicationHistorySection: React.FC<ReplicationHistorySectionProps>
     return (
       <div className="mco-dashboard__contentColumn">
         <div className="mco-dashboard__contentRow mco-cluster-app__contentRow--spaceBetween">
-          <div className="mco-dashboard__title mco-cluster-app__contentRow--flexStart">
+          <div className="mco-dashboard__title mco-cluster-app__contentRow--flexStart pf-c-title pf-m-lg">
```

Comment:
don;t we want to do the same that we are doing at other sections of the card ??
```suggestion
          <div className="mco-dashboard__title mco-cluster-app__contentRow--flexStart mco-dashboard__headerText--size">
```

Reply:
or,
```suggestion
          <div className="mco-dashboard__title mco-cluster-app__contentRow--flexStart mco-cluster-app__headerText--size">
```

Reply:
pf-c-title pf-m-lg size is even bigger than headerText.
 mco-cluster-app__headerText--size is 16px, 
 pf-c-title pf-m-lg is around 18px